crates/dc-api-types/src/query.rs

```diff
@@ -16,49 +16,27 @@ pub struct Query {
     #[serde(
         rename = "aggregates",
         default,
-        with = "::serde_with::rust::double_option",
         skip_serializing_if = "Option::is_none"
     )]
-    pub aggregates: Option<Option<::std::collections::HashMap<String, crate::Aggregate>>>,
+    pub aggregates: Option<::std::collections::HashMap<String, crate::Aggregate>>,
```

Comment:
Flattening these double-option wrappings was not strictly necessary. But the double wrapping doesn't do anything for us, and I got tired of dealing with it.